apps/web/package.json

```diff
@@ -26,7 +26,7 @@
     "cookie": "0.4.2",
     "formik": "2.2.9",
     "keycloak-js": "16.1.1",
-    "next": "12.0.4",
+    "next": "12.1.5",
```

Comment:
I tried this but it caused a lot of problems.

Reply:
Yeah, this looks to be the cause of the broken tests. 2.1.4 seems to be stable.